data_dir/cdc_profile.yaml

```diff
@@ -0,0 +1,51 @@
+keyspace: cdc_test
+
+keyspace_definition: |
+
+  CREATE KEYSPACE cdc_test WITH replication = {'class': 'SimpleStrategy', 'replication_factor': '3'}  AND durable_writes = true;
+
+table: test_table
+
+table_definition: |
+
+  CREATE TABLE cdc_test.test_table (
+    pkid text PRIMARY KEY,
+    name text,
+    number int,
+    starttime timestamp,
+    weight decimal,
+    users list<text>,
+  ) WITH bloom_filter_fp_chance = 0.01
+    AND cdc = {'enabled': true}
```

Comment:
Is this the only thing needed for CDC?
What about the pre-image option?

Reply:
The other parameters are options, 

- preimage: false by default
- postimage: currently not available 
- ttl: 86400 by default (24 hours)

@roydahan Do you expect it configurable? or we should assign some special value for them here?

Reply:
How about `cdc = {'enabled': true, 'preimage': true, 'ttl': 600}` ?

Reply:
for pre-image we should configure another longevity, because in this mode we got additional reads before insert to scylla_cdc_log

Reply:
> for pre-image we should configure another longevity, because in this mode we got additional reads before insert to scylla_cdc_log

Can we use two different workloads in same longevity job?  one is `preimage: true`, another is `preimage: false`

Reply:
Yes, why not?
Add two workloads, two different tables